src/main/java/global/govstack/usct/configuration/SecurityConfig.java

```diff
@@ -253,10 +253,10 @@ private static Collection<? extends GrantedAuthority> mapAuthorities(
         log.info("Logging in user {}", subject);
         return switch (subject) {
           case "299950323465436931629862208523254959" -> Set.of(
-              new SimpleGrantedAuthority("ROLE_" + ENROLLMENT_OFFICER));
```

Comment:
I think those changes are not needed

Reply:
I tried to follow the logic that we have in the documentation.
1. Create candidate
2. Enroll 
3. Payment 

`Copy/paste` from the documentation.